resources/leiningen/new/kraken/queue.clj

```diff
@@ -26,18 +27,30 @@
               (recur (inc attempt)))
           (do (log/error "Connecting to RabbitMQ failed. Bailing.")
               (throw (ex-info "Connecting to RabbitMQ failed" {:attemts attempt}))))))
-    (let [ok-ch (lch/open @connection)]
-      (lq/declare ok-ch
-                  "{{name}}.ok"
-                  (config :rabbitmq :queues "{{name}}.ok"))
-      (k/responder ok-ch "{{name}}.ok" handlers/ok)
+    (let [events-exchange "events"
+          events-ch (lch/open @connection)
+          heartbeat-ch (lch/open @connection)
+          ping-ch (lch/open @connection)
+          heartbeat (async/chan)]
+      (async/go-loop []
```

Comment:
Does this really belong in the queue initialize? I like it, just maybe not here.